src/entities/activity/lib/types/item.ts

```diff
@@ -106,6 +107,13 @@ export interface TextItem extends ActivityItemBase {
   answer: DefaultAnswer;
 }
 
+export interface ParagraphTextItem extends ActivityItemBase {
+  responseType: 'paragraphText';
+  config: TextItemConfig;
```

Comment:
You're reusing `TextItemConfig` for ParagraphItem but not all values will be available, only `responseRequired` and  `maxResponseLength`. You should add its own `ParagraphItemConfig`